src/field_theory/normal.lean

```diff
@@ -263,6 +263,27 @@ alg_equiv.ext (λ _, (algebra_map E K₃).injective
 def alg_equiv.restrict_normal_hom [normal F E] : (K₁ ≃ₐ[F] K₁) →* (E ≃ₐ[F] E) :=
 monoid_hom.mk' (λ χ, χ.restrict_normal E) (λ ω χ, (χ.restrict_normal_trans ω E))
 
+
+variables (F K E) [algebra K E] [is_scalar_tower F K E]
+
+/-- If `E/K/F` is a tower of fields with `K/F` normal then we have an equivalence
+ (K →ₐ[F] E) ≃ (K ≃ₐ[F] K). -/
+def normal.alg_hom_equiv_aut [normal F K] : (K →ₐ[F] E) ≃ (K ≃ₐ[F] K) :=
```

Comment:
Can you swap the variables around to match what's used in the lemmas immediately above?

```suggestion
def normal.alg_hom_equiv_aut [normal F E] : (E →ₐ[F] K₁) ≃ (E ≃ₐ[F] E) :=
```
That should remove the need for the `variables` line.